packages/web-react/src/types/modalComposed.ts

```diff
@@ -0,0 +1,56 @@
+import { ElementType, DetailedHTMLProps, HTMLAttributes, ComponentPropsWithRef } from 'react';
+import { ChildrenProps, StyleProps, ClickEvent } from './shared';
+
+type OmittedComponentStyleProps = 'style' | 'className';
```

Comment:
Ok, so I modified it like that, but in the second case the type was not enough, because I need Omit with other types, so I expanded it with those types.
`
export type OmittedComponentStyleProps<E> = Omit<E, UnsafeStyleProps>;
export type OmittedExtendedComponentStyleProps<E, P> = Omit<E, UnsafeStyleProps & P>;
`
That's why I put the mob there, where one will throw only the Unsafe ones and the other expanded one, which will have the option to throw other props as well.